Website/src/rendering/src/components/Vendors/FeaturedVendors.tsx

```diff
@@ -12,6 +19,13 @@ type FeaturedVendorsProps = ComponentProps & {
 };
 
 const FeaturedVendors = (props: FeaturedVendorsProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasVendors = props?.fields?.Vendors.length;
```

Comment:
`hasVendors` is an integer or `undefined` with that code. It could even throw if `Vendors` is undefined. We could use `!!props?.fields?.Vendors?.length` for it to always be a boolean.